app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt

```diff
@@ -201,8 +204,9 @@ class ConversationActivityV2 : PassphraseRequiredActionBarActivity(), InputBarDe
     OnReactionSelectedListener, ReactWithAnyEmojiDialogFragment.Callback, ReactionsDialogFragment.Callback,
     ConversationMenuHelper.ConversationMenuListener {
 
-    private var binding: ActivityConversationV2Binding? = null
-
+    private var _binding: ActivityConversationV2Binding? = null
+    private val binding get() = _binding!!
```

Comment:
it's perhaps preferable to use `lateinit var` which was made for this kinda stuff.

```
private lateinit var binding: ActivityConversationV2Binding
```

and that one doesn't need a default value. There's also something like `binding.isInitialized()` that is available to call to be certain, but you'd rarely if ever actually want to use it.

Reply:
Yeah you can switch to lateinit var. I can't remember which use case now because I haven't had to use view binding in so long, but I remember some cases breaking because of it hence the above. 
But these could have been worked out by now. Happy to go with a lateinit as long as we can make sure it never breaks ( again, sorry I can't remember what the use case(s) was(were)

Reply:
Nice. Replaced.